cirq/study/trial_result.py

```diff
@@ -76,11 +76,6 @@ class TrialResult:
 
     Attributes:
         params: A ParamResolver of settings used when sampling result.
-        measurements: A dictionary from measurement gate key to measurement
```

Comment:
Why was this removed? Was it moved somewhere else?

Reply:
It is documented in the Args section of `__init__`.
The reason it had to be removed is that it caused a conflict with the property `measurements` defined further down.  This was the reason for `Cirq/cirq/study/trial_result.py:docstring of cirq.TrialResult.measurements:1: WARNING: duplicate object description of cirq.TrialResult.measurements, other instance in Cirq/docs/generated/cirq.TrialResult.rst, use :noindex: for one of them`

Reply:
Perhaps it could or should be documented in the property method instead.

Reply:
Ah. Yeah, this seems like it's redundant.